README.md

```diff
@@ -53,10 +53,30 @@ To build and run, use the included Gradle wrapper script on a Unix System:
 
 On Windows:
 
-    gradlew.bat :ui:run
+    gradlew :ui:run
 
 If you don't have an internet connection you can build using the `--offline` flag if you have built GRIP once before.
 
+### Running Tests
+
+To run tests on grip, use the included Gradle wrapper script on a Unix System:
+
+    ./gradlew test
+
+On Windows:
+
+    gradlew test
+
+### Checking Style
```

Comment:
I'd prefer to see something on running the `check` task, since that will run all the code checks (checkstyle, pmd, spotbugs), as well as running all the tests. This matches what's run on CI, which is helpful for doing local testing before pushing and being surprised by things breaking on CI